src/web-components.js

```diff
@@ -48,6 +48,26 @@ export function isShadowDomSupported() {
 }
 
 /**
+ * Returns `true` if Shadow CSS encapsulation is supported.
+ * @return {boolean}
+ */
+export function isShadowCssSupported() {
+  if (!isShadowDomSupported()) {
+    return false;
+  }
+
+  //TODO: Find a better way to test CSS encapsulation
+  if (window.ShadyCSS) {
```

Comment:
Yes I am excited to tackle this as well, I will probably start diving into it after I finish up my conversion interviews